msfvenom

```diff
@@ -222,7 +197,7 @@ def parse_args(args)
     opts[:payload] = "stdin"
   end
 
-  if opts[:payload] == 'stdin' and not opts[:list]
+  if opts[:payload].downcase == 'stdin' and not opts[:list]
```

Comment:
`&&` is preferred over `and`

Reply:
Thanks!
Found another place to alter (as well as a not!)